packages/ember-cli-code-coverage/addon-test-support/index.js

```diff
@@ -0,0 +1,27 @@
+let __require;
```

Comment:
Can you add a bit of documentation to the top of this file explaining the conceptual thing that it is doing?